packages/rancher-logging/overlay/templates/userroles.yaml

```diff
@@ -15,31 +15,20 @@ rules:
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
-metadata:
-  name: "logging-edit"
-  labels:
-    rbac.authorization.k8s.io/aggregate-to-edit: "true"
-rules:
-  - apiGroups:
-      - "logging.banzaicloud.io"
-    resources:
-      - flows
-      - outputs
-    verbs:
-      - "*"
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
 metadata:
   name: "logging-view"
   labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
     rbac.authorization.k8s.io/aggregate-to-view: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
 rules:
   - apiGroups:
       - "logging.banzaicloud.io"
     resources:
       - flows
       - outputs
+      - clusterflows
+      - clusteroutputs
```

Comment:
If we are giving view to all users, is there any sensitive info they could see on any 4 of these CRD's ? Or are they all in secrets.

Reply:
It is possible to store credentials/certificates as string values in the (Cluster)Outputs YAMLs. Although we're mounting from secrets using the UI.